maskes/settings.py

```diff
@@ -27,9 +21,9 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = env.str('SECRET_KEY')
+SECRET_KEY = 'THIS-IS-A-DUMMY-KEY'
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = env('DEBUG')
+DEBUG = True
```

Comment:
This makes me nervous based on the warning in the comment above. Can we set this to true unless the `DEBUG` environment variable is set?